tests/test_flow_decomposition.py

```diff
@@ -93,6 +115,8 @@ def test_flow_decomposition_parameters():
     logging.getLogger('powsybl').setLevel(logging.DEBUG)
     test_demo()
     print('\n\n\n\n')
+    test_demo_one_by_one()
+    print('\n\n\n\n')
```

Comment:
a little thing could you remove all the print in this file ?

Reply:
done